recipes/ddtrace/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ddtrace" %}
+{% set version = "0.42.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ddtrace-{{ version }}.tar.gz
+  sha256: dba66c7013a4352dfa20ed31e989001250056c89811d1cb5ee10e23dc98bac0a
+
+build:
+  number: 0
+  skip: true
```

Comment:
This line prevents the package from building at all, and probably should be removed

Reply:
Thanks for the feedback! This one made me a little comfortable. This was autogenerated via `grayskull`, I'll update.

Reply:
You should let grayskull know. That seems like a bug

Reply:
Looks like the selectors got removed when the comments were removed.  We should probably update the bot to state that selector lines are not comments?

Reply:
Right, this was me removing the "comment," which it seems is not just a cosmetic change. Maybe the information that's currently being captured via those comments can instead be part of the `yaml` longterm?